unitTests/CMakeLists.txt

```diff
@@ -14,13 +31,24 @@ add_executable(unit_tests mainTestRunner.cpp
     test_helper_functions.cpp
     test_slice.cpp
     test_image_int.cpp
-    $<TARGET_OBJECTS:exiv2lib_int>
+    ${unit_tests_exiv2lib_SOURCES}
```

Comment:
I would not change this. The idea of using the OBJECT library is to not recompile things. With a CMake OBJECT library we just take the .o files compiled while generating `exiv2lib` and we reuse them in the test target. With this change, we would be compiling all these files twice.

Reply:
The reason of doing so is that Windows seems to distinguish between normal symbols and DLL-imported symbols. If I just take the object files of `exiv2lib_int` and `exiv2lib` and put them here, Visual Studio complains that symbols it expects to be provided by a DLL ends up being linked statically. Therefore, I use `exiv2lib_STATIC` macro to disable `dllimport` and `dllexport` attributes and compile the same source files again in order to generate object files which can be linked statically.

Reply:
oh I see. Thanks for the feedback and the efforts for dealing with this issue :+1:

Reply:
@lantw44  I support @piponazo in thanking you for dealing with this issue.  You (and @kevinbackhouse) are the best contributors outside of Team Exiv2 in 2019.  Thank you very much.

When I write the release notes, I like to acknowledge contributors by name.  Here's the 

*Exiv2 v0.27.2.2 = Exiv2 v0.27.2 RC2*

_Second Release Candidate for quarterly “dot” release of Exiv2 v0.27.2_

A huge thank you to Kevin for outstanding contributions to this release.
Thank You to Dan and Luis for their solid efforts on Exiv2.
Thank You to Andreas S for representing Team Exiv2 at LGM 2019.
Contributors: Jens, Nehal, Phil, Toni

I'd like to give **YOU** a _**"huge thank you also"**_ in v0.27.2 if you'd like to tell me your name!

@phako: You'll see I have already acknowledged you.  Thank You also for your contribution.